website/server/libs/inbox/index.js

```diff
@@ -141,6 +142,7 @@ export async function listConversations (owner) {
   const conversations = conversationsList.map(res => ({
     uuid: res._id,
     ...res,
+    backer: Array.isArray(res.npc) && res.npc.length > 0 ? { npc: res.npc[0] } : { npc: null },
```

Comment:
couldn't `usersMap[res._id].backer` be used here like it's used for `contributor`? By adding backer here https://github.com/HabitRPG/habitica/pull/11647/files#diff-ebb5855000db09391acfb6b15aed8fc5R96 ?

Reply:
yeah adding `backer: { $last: '$backer' },` to line 97 and then also here as mentioned `usersMap[res._id].backer` should be enough , then the changes in line 128 and 145 are obsolete